File: Models/Movie.cs
using System;
using System.Collections.Generic;

namespace SIVU_OLIVIU_ID_IE_PROIECT_MP.Models
{
    public class Movie
    {
        public Movie()
        {
            this.Genres = new List<Genre>();
            this.Cast = new List<Actor>();
        }

        public int Id { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public DateTime ReleaseDate { get; set; }

        public int Rating { get; set; }

        public virtual ICollection<Genre>  Genres { get; set; }

        public virtual ICollection<Actor> Cast { get; set; }

        public Director Director { get; set; }
    }
}

File: ClientApp/src/components/home/Movie.js
import React, { useEffect, useState } from "react";

const Movie = (props) => {
  const [cast, setCast] = useState("");
  const [genre, setGenre] = useState("");

  useEffect(() => {
    setCast(props.cast.join());
    setGenre(props.genres.join());
  }, []);

  return (
    <div className="movie mr-20">
      <img src={`${props.imgPath}`} />
      <div className="">
        <ul>
          <li>
            <b>{props.title}</b>
          </li>
          <li className="mt-3">
            <b>Director:</b> {props.director}
          </li>
          <li>
            <b>Cast:</b> {cast}
          </li>
          <li>
            <b>Genre:</b> {genre}
          </li>
          <li>
            <b>Rating:</b> {props.rating}
          </li>
        </ul>
      </div>
    </div>
  );
};

export default Movie;
